draft-ietf-quic-transport.md

```diff
@@ -1468,10 +1468,11 @@ magnitude of any amplification attack that can be mounted using spoofed source
 addresses.  In determining this limit, servers only count the size of
 successfully processed packets.
 
-Clients MUST pad UDP datagrams that contain only Initial packets to at least
-1200 bytes.  Once a client has received an acknowledgment for a Handshake packet
-it MAY send smaller datagrams.  Sending padded datagrams ensures that the server
-is not overly constrained by the amplification restriction.
+Clients MUST ensure that UDP datagrams containing Initial packets are sized to
+at least 1200 bytes, padding packets in the datagram if necessary.  Once a
+client has received an acknowledgment for a Handshake packet it MAY send smaller
```

Comment:
To me this looks like an outdated text prior to draft-12.

Now that the server responds with an Initial packet, my understanding is that a client in required to send full-sized datagrams until it receives an _Initial_ packet from the server.

Reply:
I think we should stick with the simplest text possible and we should remove the second sentence.  Once you've received an ACK for a handshake packet, you should be done sending INITIAL packets anyway, so it feels like an unnecessary optimization.

Reply:
Agreed with @ianswett since the time when you receive the ACK for the Handshake packet is exactly when you stop sending Initial frames, so this sentence feels like its pointing out a subtlety that doesn't exist.

If the point of the sentence is to say that, while UDP datagrams containing Initial packets MUST be >= 1200 bytes, all others may or may not be padded (at the implementation's discretion), it would be clearer to say:

`Any UDP datagram that does not contain any Initial packets MAY be smaller than 1200 bytes.`